change/office-ui-fabric-react-2019-08-01-14-49-02-fix-duplicate-export.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
Wouldn't this be at least a `minor` change?

Reply:
This should be backwards compatible, as `SelectionZone` is not directly importable. Why would it be minor instead of patch?

Reply:
Nevermind, it makes sense since it is not directly importable. I'll mark this comment as resolved.